packages/dds/sequence/src/sequence.ts

```diff
@@ -485,6 +497,44 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 		return this.intervalCollections.get(label);
 	}
 
+	/**
+	 * The collection of API's to track the local reference of the associated client
+	 */
+	public get intervalCollectionClient(): LocalReferenceTracker {
+		return {
+			createLocalReferencePosition: (
```

Comment:
this class already has a public method, createLocalReferencePosition, we should just use that

Reply:
all these method in fact already exist this class already, no reason to proxy to the client again